cql/AutogeneratedTableLookup.cql

```diff
@@ -184,6 +184,9 @@ define function GetSurveillanceManagement(prev2HpvVal String, prev2PapVal String
     }
 
 define function GetRowTable2(prev2HpvVal String, prev2PapVal String, prevHpvVal String, prevPapVal String, currHpvVal String, currPapVal String):
+  _GetRowTable2(AnyHpv(prev2HpvVal), prev2PapVal, AnyHpv(prevHpvVal), prevPapVal, currHpvVal, currPapVal)
```

Comment:
- Does `GetAdjacentRowsTable2` function also need `AnyHpv()` call?
- Does `GetAdjacentRowsTable5` function also need `AnyHpv()` call?

Reply:
Added appropriate AnyHpv() calls to GetAdjacentRowsTable2. GetAdjacentRowsTable5 doesn't need AnyHpv() because the LongTermSurveillance function is only looking for associated HPV-negative values.

Reply:
I also added test resources to the ManagementTable2 tests to iterate over HPV+, HPV16+ and HPV16-,18+ results in the test case histories to exercise finding the adjacent rows in table 2.

Reply:
The added `AnyHPV()` calls to `GetAdjacentRowsTable2` look good.

Reply:
I see why we currently wouldn't need the `AnyHPV()` call for Table 5. But do you know why the GetRowTable5 function only looks for the scenarios where the HPV input is one of the multiple-negative-test scenarios? This is technically not directly related to this PR, so shouldn't hold this up from being merged, but it could be worth some investigation in the future.

Reply:
I opened [CCSMCDS-400](https://jira.mitre.org/browse/CCSMCDS-400) to prompt an investigation about the GetAdjacentRowsTableX logic. @mrnosal if you have already done this investigation, feel free to close the Jira ticket, or move it into the current sprint as you see fit.

Reply:
I see - this will require additional investigation. "AdjacentRows" is our concept, so the behavior is determined by us.